examples/Classification/pointcloud_classification.py

```diff
@@ -16,28 +17,43 @@
 parser.add_argument('--epochs', type=int, default=10, help='Number of train epochs.')
 parser.add_argument('-lr', '--learning-rate', type=float, default=1e-3, help='Learning rate.')
 parser.add_argument('--batch-size', type=int, default=12, help='Batch size.')
-parser.add_argument('--device', type=str, default='cuda', help='Device to use.')
+parser.add_argument('--viz-test', action='store_true', help='Visualize an output of a test sample')
+parser.add_argument('--transforms-device', type=str, default='cuda', help='Device to use.')
 
 args = parser.parse_args()
 
 
+def to_device(inp):
+    inp.to(args.transforms_device)
+    return inp
+
 transform = tfs.Compose([
+    to_device,
     tfs.TriangleMeshToPointCloud(num_samples=args.num_points),
     tfs.NormalizePointCloud()
 ])
 
+if args.transforms_device == 'cuda':
+    num_workers = 0
+    pin_memory = False
+else:
+    num_workers = cpu_count()
```

Comment:
Might not be a good idea to use all cpus? Like in pytorch examples, let's set the default to a nominal value (like `1` or `2`) and let the user decide on the number of threads to spawn?

cf. https://github.com/pytorch/examples/blob/master/mnist/main.py#L100

Reply:
This strongly depend on the training loop bottleneck: 
https://github.com/pytorch/examples/blob/master/dcgan/main.py#L19
https://github.com/pytorch/examples/blob/master/imagenet/main.py#L33
https://github.com/pytorch/examples/blob/master/super_resolution/main.py#L20

Our conversion function right now is just not efficient enough on CPU to be interesting with such a low number of workers, we will be working on that later